### PR TITLE
OpenAPI and UAT examples for outbound deductions

### PR DESCRIPTION
**Pull request by mattd-kainos**

### CODE REVIEW

File: specification/nhais-adaptor.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ servers:
 tags:
```
Comment (mattd-kainos) [inline specification/nhais-adaptor.yaml:48]: Need to update HTML version once Amendment PR is merged

File: specification/nhais-adaptor.yaml [hunk 2]
```diff
@@ -61,19 +59,49 @@ paths:
 
         Use this endpoint to send a Deduction transaction to NHAIS.
 
-        ## Operation Id
+        ## Supported Fields
 
-        When you submit an amendment an operation id will be included in the response. This operation id MUST be retained
-        to match against the asynchronous reply from NHAIS
+        Refer to GP Links Specification Chapter 3 for the complete requirements for mantually-entered acceptances. The
```
Comment (MattLewis80) [inline specification/nhais-adaptor.yaml]: mantually?

File: specification/nhais-adaptor.yaml [hunk 3]
```diff
@@ -170,20 +196,26 @@ paths:
...
         https://digital.nhs.uk/services/nhais/guide-to-nhais-gp-links-documentation
 
-        Three fields are deprecated and not supported by the adaptor:
+        Three fields are DEPRECATED and not supported by the adaptor:
 
         * RPP Mileage
         * Blocked Route/Special District Marker
         * Walking Units
 
+        REQUIRED - If any of these data items are not provided or are blank then the adaptor will produce a 400 error
```
Comment (MattLewis80) [inline specification/nhais-adaptor.yaml:205]: *REQUIRED - THE ONLY FIELDS THAT ARE VALIDATED BY THE ADAPTOR. If any of these data items are not provided or are blank then the adaptor will produce a 400 error

File: specification/nhais-adaptor.yaml [hunk 4]
```diff
@@ -170,20 +196,26 @@ paths:
...
+        OPTIONAL - These data items may be included if available
+        DEPRECATED - The adaptor provides no FHIR representation for these items and it is not possible to include them
+        NOT REQUIRED - These data items are will be ignored if provided
 
+        Any other values not listed in the tables below and provided in the request that are valid for a FHIR R4
+        Parameter or Patient resource will be ignored.
```
Comment (MattLewis80) [inline specification/nhais-adaptor.yaml:211]: GP Suppliers should consider adding their own validation around 'OPTIONAL', 'DEPRECATED' and 'NOT REQUIRED' field types as NO VALIDATION IS APPLIED BY THE ADAPTOR.

File: specification/nhais-adaptor.yaml (hunk 3 above)
Comment (mattd-kainos) [reply specification/nhais-adaptor.yaml:205]: as discussed in a separate PR we'll add a section to the top-level description about input validation that covers all transaction types

File: specification/nhais-adaptor.yaml (hunk 4 above)
Comment (mattd-kainos) [reply specification/nhais-adaptor.yaml:211]: as discussed in a separate PR we'll add a section to the top-level description about input validation that covers all transaction types

**Review by jaroslaw-wojciechowski — APPROVED** [review]

**Pull request merged**